plugins/vendor/chargeBee/lib/api/types/KalturaChargeBeeVendorIntegration.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * @package plugins.chargeBee
+ * @subpackage api.objects
+ */
+class KalturaChargeBeeVendorIntegration extends KalturaObject implements IFilterable
+{
+	/**
+	 * @var int
+	 * @readonly
+	 * @filter eq,in
+	 */
+	public $id;
+
+	/**
+	 * @var time
+	 */
+	public $lastAccessToChargeBee;
+
+	/**
+	 * @var string
+	 */
+	public $handledEventIds;
+
+	/**
+	 * @var KalturaVendorTypeEnum
+	 * @filter eq,in
+	 */
+	public $type;
+
+	/**
+	 * @var string
+	 * @filter eq
+	 */
+	public $subscriptionId;
+
+	/**
+	 * @var KalturaVendorStatus
+	 * @filter eq,in
+	 */
+	public $status;
+
+	/**
+	 * @var time
+	 * @readonly
+	 * @filter gte,lte,order
+	 */
+	public $createdAt;
+
+	/**
+	 * @var time
+	 * @readonly
+	 * @filter gte,lte,order
+	 */
+	public $updatedAt;
+
+	/**
+	 * @var int
+	 * @filter eq
+	 */
```

Comment:
Should be insert only to avoid update of this value

Reply:
I think read-only only is more sutibale for the partnerId field